code/modules/clothing/spacesuits/void/station.dm

```diff
@@ -18,9 +18,10 @@
 	item_state = "eng_voidsuit"
 	slowdown = 1
 	armor = list(melee = 40, bullet = 35, laser = 35,energy = 5, bomb = 35, bio = 100, rad = 90)
-	allowed = list(
-		/obj/item/device/lighting/toggleable/flashlight,/obj/item/weapon/tank,/obj/item/device/suit_cooling_unit,
-		/obj/item/weapon/storage/bag/ore,/obj/item/device/t_scanner,/obj/item/weapon/tool/pickaxe,
+	extra_allowed = list(
+		/obj/item/weapon/storage/toolbox,
```

Comment:
Toolbox in suit storage slot?

Reply:
I didn't change this, engineering voidsuits already allowed it.

People never really carry toolboxes around otherwise, so i don't see the harm